src/containers/ConfigComputeResource.tsx

```diff
@@ -14,23 +18,21 @@ interface OwnProps {
 
 type Props = StateProps & DispatchProps & OwnProps
 
-const  ComponentName: FunctionComponent<Props> = ({ nodeId }) => {
-    const Content = () => {
-        return <div>
-            <pre>kachery-p2p node ID: {nodeId}</pre>
-        </div>
+const ConfigComputeResource: FunctionComponent<Props> = ({ serverInfo }) => {
+    const substitute = {
+        nodeId: serverInfo.nodeId,
+        computeResourceUri: (serverInfo?.labboxConfig)?.compute_resource_uri || 'local'
     }
-
     return (
-        <div>
-            <h1>Compute resource configuration</h1>
-            <Content />
-        </div>
+        <Markdown
+            source={md}
+            substitute={substitute}
```

Comment:
Convenience: pass substitute object into markdown (so we can use the markdown as a template file)